### PR TITLE
Sort pressed content by date

### PR DESCRIPTION
**Pull request by ioannakok**
## What does this change?
It sorts the pressed content by date.

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No
- [ ] Yes (please indicate your plans for DCR Implementation)

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [x] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [x] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [x] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/main/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: common/app/services/dotcomrendering/PressedContent.scala [hunk 1]
```diff
@@ -9,223 +9,227 @@ object PressedContent {
...
+    "/environment/2015/oct/13/agriculture-farming-food-addiction-meat-harvest-hungry-world",
+    "/us-news/2015/nov/05/police-tasers-deaths-the-counted",
+    "/environment/ng-interactive/2015/nov/26/the-mekong-river-stories-from-the-heart-of-the-climate-crisis-interactive",
+    "/us-news/2015/dec/01/the-county-kern-county-deadliest-police-killings",
+    "/us-news/2015/dec/04/the-county-kern-county-california-deputies-tactics",
+    "/us-news/2015/dec/08/the-county-kern-county-california-sexual-assault-secret-payoffs",
+    "/us-news/2015/dec/10/kern-county-california-police-killings-misconduct-district-attorney",
+    "/us-news/2015/dec/31/ties-that-bind-conflicts-of-interest-police-killings",
+    // 2016
+    "/technology/2016/apr/12/the-dark-side-of-guardian-comments",
```
Comment (AshCorr) [inline common/app/services/dotcomrendering/PressedContent.scala:66]: Duplicate (but it duplicated before your PR)

**Review by AshCorr — APPROVED** [review]
Thanks for tidying this up :)

File: common/app/services/dotcomrendering/PressedContent.scala [hunk 2]
```diff
@@ -9,223 +9,227 @@ object PressedContent {
...
+    "/world/2016/sep/24/americas-war-the-killing-jaiden-dixon-and-tyler-dunn",
+    "/cities/ng-interactive/2016/nov/10/subterranean-london",
+    "/music/2016/nov/11/the-future-50-the-rising-stars-to-look-out-for",
+    "/science/2016/nov/21/magic-numbers-can-maths-equations-be-beautiful",
+    "/film/2016/nov/29/the-50-best-uk-films-of-2016-full-list",
+    "/film/2016/nov/29/50-best-us-film-of-2016-the-full-list",
+    "/music/2016/nov/30/the-best-albums-of-2016",
+    "/tv-and-radio/2016/dec/06/best-tv-shows-of-2016",
+    "/info/ng-interactive/2016/dec/07/sign-up-for-the-flyer",
+    "/tv-and-radio/2016/dec/17/christmas-tv-2016-50-shows-to-watch",
```
Comment (AshCorr) [inline common/app/services/dotcomrendering/PressedContent.scala:89]: Also duplicated

**Review by AshCorr — APPROVED** [review]

**Review by mxdvl — APPROVED** [review]
Much easier to parse!

File: common/app/services/dotcomrendering/PressedContent.scala (hunk 2 above)
Comment (ioannakok) [reply common/app/services/dotcomrendering/PressedContent.scala:89]: Good catch Ash! Thanks!

**Pull request merged**